yt_dlp/extractor/telegram.py

```diff
@@ -18,8 +18,9 @@ class TelegramEmbedIE(InfoExtractor):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        webpage = self._download_webpage(url, video_id, query={'embed': 0})
-        webpage_embed = self._download_webpage(url, video_id, query={'embed': 1}, note='Downloading ermbed page')
+        is_single = ("?single" in url);
```

Comment:
Semicolons are discouraged in Python, also use single quotes.
```suggestion
        is_single = '?single' in url
```
It might make sense to properly parse the querystring instead using `utils.parse_qs` instead.
```suggestion
        is_single = 'single' in parse_qs(url)
```